src/main/java/org/folio/consortia/domain/dto/Type.java

```diff
@@ -0,0 +1,5 @@
+package org.folio.consortia.domain.dto;
+
+public enum Type {
```

Comment:
There can be different types in classpath for different domains, lets rename to CustomFieldType